config.rb

```diff
@@ -11,6 +11,7 @@
       { :assume_extension => true,
         :disable_external => true,
         :allow_hash_href => true,
+        :empty_alt_ignore => true,
```

Comment:
It'd be good to understand why this has been added – could this be a separate commit, or mentioned in the commit message?

Reply:
Hi @36degrees - Good point, sorry, should have mentioned this! `html-proofer` fails the build fails because of the missing alt tags. I believe this isn't technically a fail because the images are either SVG or explained in the surrounding content. 

However, I've realised we should add `alt=""` (or as close as we can get to it in TDT, which is `alt` alone) to the images. So I'll add that in a better-described commit.

Reply:
The version of html-proofer hasn't changed in this pull request, so what's changed that means we didn't need to do this before?

If this is unrelated to the dependency updates – if the build is currently failing on `master` – then the cleanest thing to do would probably be to raise a separate pull request that fixes the build, then rebase this once that's been merged.

Reply:
This is a good question and led to some digging!

There's a change somewhere that's changed the way the `img` is tag is rendered when there's no provided alt tag.

- Originally the alt tag would use the filename, which is an html-proofer pass. So the current master passes.
- In 2.0.12 of the TDT, the alt tag becomes just `alt`, which is an html-proofer fail. So this PR build fails.

So I think we do need to update html-proofer's options simultaneously with updating to 2.0.12? I've tweaked the commits to separate things out more cleanly though.

Let me know if I've misunderstood anything!